DOCUMENTATION.md

```diff
@@ -129,7 +131,8 @@ This section will let you know if the selected remote is not valid or if the pus
 ### Git Difference
 
 In "Git Difference" click on the modified file to see the difference. The platform has syntax highlighting available for a limited set of [languages](LANGUAGES.md)
-![git-difference](https://user-images.githubusercontent.com/65342122/87243040-11021b80-c450-11ea-8775-d52dcc7f57e1.png)
+
+![git-difference](https://user-images.githubusercontent.com/65342122/89164179-608dc000-d594-11ea-8e00-a880b2bdcc88.png)
 
```

Comment:
The loading stages UI in fetch, pull and push views have been updated, but the push to remote still holds the old loading image